pkg/apis/argoproj/v1alpha1/argocd_types.go

```diff
@@ -419,6 +424,9 @@ type ArgoCDSpec struct {
 	// Server defines the options for the ArgoCD Server component.
 	Server ArgoCDServerSpec `json:"server,omitempty"`
 
+	// SSO defines the user management for Argo CD
```

Comment:
(Comments will go to the generated CRD as field documentation.)  SSO and user management are quite different.   Let's put more descriptive comment.

It should not be a pointer.